src/apiClient.ts

```diff
@@ -161,37 +180,44 @@ export default function APIClient(
         }
     };
 
-    this.sendSingleForwardingStatsToServer = function(forwardingStatsData) {
-        let url;
-        let data;
-        try {
-            const xhrCallback = function() {
-                if (xhr.readyState === 4) {
-                    if (xhr.status === 202) {
-                        mpInstance.Logger.verbose(
-                            'Successfully sent  ' +
-                                xhr.statusText +
-                                ' from server'
-                        );
-                    }
-                }
-            };
-            const xhr = mpInstance._Helpers.createXHR(xhrCallback);
-            url = mpInstance._Helpers.createServiceUrl(
-                mpInstance._Store.SDKConfig.v1SecureServiceUrl,
-                mpInstance._Store.devToken
-            );
-            data = forwardingStatsData;
+    this.sendSingleForwardingStatsToServer = async (
+        forwardingStatsData: IForwardingStatsData
+    ) => {
```

Comment:
```suggestion
    this.sendSingleForwardingStatsToServer = async (
        forwardingStatsData: IForwardingStatsData
    ): void => {
```

Reply:
i ended up moving this to the forwarder.js file, so we lost the typing here.  it seemed to fit better there logically, but i can revert if you think keeping the types is more important than organization